src/crewai/llm.py

```diff
@@ -153,7 +153,13 @@ def call(self, messages: List[Dict[str, str]], callbacks: List[Any] = []) -> str
                 params = {k: v for k, v in params.items() if v is not None}
 
                 response = litellm.completion(**params)
-                return response["choices"][0]["message"]["content"]
+                if params.get("stream", False):
```

Comment:
If you're trying to listen to the stream, wouldn't you want this to be `True`?

Reply:
"stream is set to False in params by default but can be overridden through kwargs. 
Despite this, here we pass a default value in get() function, to make the code more readable and independent of preceding code. 
So, stream option can be enabled by passing stream=True when creating crewai.LLM instance.